.vale.ini

```diff
@@ -0,0 +1,13 @@
+StylesPath = .github/styles
+MinAlertLevel = suggestion
+Vocab = Keptn
+
+[*.{md,mdx}]
```

Comment:
:memo: Apply the style suggestions/checks only on files with  `.md` and `.mdx` extension

```
[*]
# Format-specific settings appear
# under a user-provided "glob"
# pattern.
```

https://vale.sh/docs/topics/config